incubator/hnc/pkg/kubectl/set.go

```diff
@@ -130,8 +166,9 @@ func normalizeStringArray(in []string) []string {
 }
 
 func newSetCmd() *cobra.Command {
-	setCmd.Flags().Bool("root", false, "Turns namespace into root namespace")
-	setCmd.Flags().String("parent", "", "Parent namespace")
-	setCmd.Flags().Bool("allowCascadingDelete", false, "Allows cascading deletion of its self-serve subnamespaces.")
+	setCmd.Flags().BoolP("root", "r", false, "Turns namespace into root namespace")
```

Comment:
Nit not sure if it would be clearer to mention "removes current parent" for "root" flag.

Reply:
Updated the descriptions, ptal!